packages/utils/src/error-helpers/server-operations.ts

```
@@ -0,0 +1,155 @@
+      console.debug(JSON.stringify(formattedError));
+      break;
+    case "info":
+      console.info(JSON.stringify(formattedError));
+      break;
+    case "warn":
+      console.warn(JSON.stringify(formattedError));
+      break;
+    case "fatal":
+      console.error("FATAL:", JSON.stringify(formattedError));
+      break;
+    case "error":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)